### PR TITLE
Modify Chinese language translation

### PR DESCRIPTION
**Pull request by linwoodpendleton**
Provided a more detailed Chinese translation

### CODE REVIEW

**Comment by linwoodpendleton** [comment]
[Add the feature to pause record parsing.](https://github.com/poweradmin/poweradmin/pull/520/commits/c152f4b61a7193ab0b5444cb6d1b9eb99a549cbe)

**Comment by edmondas** [comment]
There should be no debugging stuff in your PR, and two PRs should be separated since they contain unrelated changes. In addition, the Chinese translation was updated by another contributor based on another PR.

File: install/helpers/database_helpers.php [hunk 1]
```diff
@@ -46,7 +46,9 @@ function updateDatabase($db, $databaseCredentials): void
 
     $fill_perm_items = $db->prepare('INSERT INTO perm_items VALUES (?, ?, ?)');
     $def_permissions = include 'includes/permissions.php';
+    echo "456";
```
Comment (edmondas) [inline install/helpers/database_helpers.php:49]: remove this

File: install/helpers/database_helpers.php [hunk 2]
```diff
@@ -46,7 +46,9 @@ function updateDatabase($db, $databaseCredentials): void
...
     $db->executeMultiple($fill_perm_items, $def_permissions);
+    echo "789";
```
Comment (edmondas) [inline install/helpers/database_helpers.php:51]: not necessary

**Pull request closed without merging**